lighthouse-core/scripts/print-contributors.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Print names and github usernames of new contributors in specified range.
+ * Ex: node lighthouse-core/scripts/print-contributors.js v6.4.0 HEAD
+ */
+
+/* eslint-disable no-console */
+
+const {execFileSync} = require('child_process');
+const fetch = require('isomorphic-fetch');
+
+const startingHash = process.argv[2];
+const endingHash = process.argv[3];
+
+async function main() {
+  const previousAuthors = execFileSync('git', [
+    '--no-pager',
+    'log',
+    '--format=%ae %aN',
+    `v2.3.0..${startingHash}`,
+  ], {encoding: 'utf8'}).trim().split('\n').map(line => {
+    const [email, name] = line.split(' ');
```

Comment:
use another delimeter to avoid splitting the first/last? or just comment if that was intentional :)